### ISSUE TITLE
Selectively Contrastive Triplet loss

### ISSUE THREAD

**Issue by littleredxh** [KevinMusgrave/pytorch-metric-learning#287]
Hi Kevin

Would you mind add my Selectively Contrastive Triplet loss which is published in ECCV2020. The major idea of this paper is to overcome the local minima during the triplet optimization. This loss work especially well on high intra-variance datasets such as Hotel50K. And I provide my implementation in this repo: 

https://github.com/littleredxh/EasyPositiveHardNegative.git

I think this loss function can be easily adapted into your framework.

**Comment by KevinMusgrave** [comment]
👍  Will add to my to-do list
